<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="3" passed="1">
  <reporter-output>
    <line>
      <![CDATA[<a href=file:///G:/Screen/screenshot_checkLogInBtnEnabled_2017-08-24_15-47.png target="_blank">screenshot_checkLogInBtnEnabled_2017-08-24_15-47</a>]]>
    </line>
    <line>
      <![CDATA[<a href=file:///G:/Screen/screenshot_checkLogInVloop_2017-08-24_15-47.png target="_blank">screenshot_checkLogInVloop_2017-08-24_15-47</a>]]>
    </line>
  </reporter-output>
  <suite name="Suite1" duration-ms="32025" started-at="2017-08-24T13:47:11Z" finished-at="2017-08-24T13:47:43Z">
    <groups>
    </groups>
    <test name="Vloop-package" duration-ms="32025" started-at="2017-08-24T13:47:11Z" finished-at="2017-08-24T13:47:43Z">
      <class name="TestsVloop">
        <test-method status="PASS" signature="createAndStartService()[pri:0, instance:TestsVloop@887af79]" name="createAndStartService" is-config="true" duration-ms="219" started-at="2017-08-24T15:47:11Z" finished-at="2017-08-24T15:47:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createAndStartService -->
        <test-method status="PASS" signature="createDriver()[pri:0, instance:TestsVloop@887af79]" name="createDriver" is-config="true" duration-ms="8170" started-at="2017-08-24T15:47:11Z" finished-at="2017-08-24T15:47:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDriver -->
        <test-method status="FAIL" signature="checkLogInBtnEnabled()[pri:0, instance:TestsVloop@887af79]" name="checkLogInBtnEnabled" duration-ms="335" started-at="2017-08-24T15:47:19Z" finished-at="2017-08-24T15:47:19Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[true expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: true expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at TestsVloop.checkLogInBtnEnabled(TestsVloop.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogInBtnEnabled -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:TestsVloop@887af79]" name="tearDown" is-config="true" duration-ms="1514" started-at="2017-08-24T15:47:19Z" finished-at="2017-08-24T15:47:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=checkLogInBtnEnabled status=FAILURE method=TestsVloop.checkLogInBtnEnabled()[pri:0, instance:TestsVloop@887af79] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<a href=file:///G:/Screen/screenshot_checkLogInBtnEnabled_2017-08-24_15-47.png target="_blank">screenshot_checkLogInBtnEnabled_2017-08-24_15-47</a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="createAndStartService()[pri:0, instance:TestsVloop@887af79]" name="createAndStartService" is-config="true" duration-ms="515" started-at="2017-08-24T15:47:21Z" finished-at="2017-08-24T15:47:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createAndStartService -->
        <test-method status="PASS" signature="createDriver()[pri:0, instance:TestsVloop@887af79]" name="createDriver" is-config="true" duration-ms="7846" started-at="2017-08-24T15:47:21Z" finished-at="2017-08-24T15:47:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDriver -->
        <test-method status="FAIL" signature="checkLogInVloop()[pri:0, instance:TestsVloop@887af79]" name="checkLogInVloop" duration-ms="2136" started-at="2017-08-24T15:47:29Z" finished-at="2017-08-24T15:47:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [http://alpha.vloop.io/hme/] but found [http://alpha.vloop.io/home/]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [http://alpha.vloop.io/hme/] but found [http://alpha.vloop.io/home/]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at TestsVloop.checkLogInVloop(TestsVloop.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogInVloop -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:TestsVloop@887af79]" name="tearDown" is-config="true" duration-ms="1160" started-at="2017-08-24T15:47:31Z" finished-at="2017-08-24T15:47:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=checkLogInVloop status=FAILURE method=TestsVloop.checkLogInVloop()[pri:0, instance:TestsVloop@887af79] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<a href=file:///G:/Screen/screenshot_checkLogInVloop_2017-08-24_15-47.png target="_blank">screenshot_checkLogInVloop_2017-08-24_15-47</a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="createAndStartService()[pri:0, instance:TestsVloop@887af79]" name="createAndStartService" is-config="true" duration-ms="503" started-at="2017-08-24T15:47:32Z" finished-at="2017-08-24T15:47:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createAndStartService -->
        <test-method status="PASS" signature="createDriver()[pri:0, instance:TestsVloop@887af79]" name="createDriver" is-config="true" duration-ms="7468" started-at="2017-08-24T15:47:33Z" finished-at="2017-08-24T15:47:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDriver -->
        <test-method status="PASS" signature="checkWrongEmail()[pri:0, instance:TestsVloop@887af79]" name="checkWrongEmail" duration-ms="1375" started-at="2017-08-24T15:47:40Z" finished-at="2017-08-24T15:47:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkWrongEmail -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:TestsVloop@887af79]" name="tearDown" is-config="true" duration-ms="761" started-at="2017-08-24T15:47:42Z" finished-at="2017-08-24T15:47:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=checkWrongEmail status=SUCCESS method=TestsVloop.checkWrongEmail()[pri:0, instance:TestsVloop@887af79] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestsVloop -->
    </test> <!-- Vloop-package -->
  </suite> <!-- Suite1 -->
</testng-results>
